test/installer/scripts/prep_omsadmin.sh

```diff
@@ -4,23 +4,25 @@ RUBY_TESTVERS=$2
 TESTDIR_SKEL=/tmp/test_omsadmin_
 TESTDIR=$TESTDIR_SKEL$$
 mkdir -p $TESTDIR
+
 echo "$TESTDIR"
 
 OMSADMIN=$TESTDIR/omsadmin.sh
 cp $BASE_DIR/installer/scripts/omsadmin.sh $OMSADMIN
 chmod u+wx $OMSADMIN
 
-sed -i s,VAR_DIR=.*,VAR_DIR=$TESTDIR,1 $OMSADMIN
```

Comment:
Existing system tests assume that the files inside certs/ conf/ and tmp/ all end up in $TESTDIR. Because of this, there were several possible ways to integrate the new omsadmin.sh variables. I chose the method of keeping the existing system tests that assume no multihoming, which meant CONF_DIR and others had to be set rather than VAR_DIR and ETC_DIR.
Perhaps a greater overhaul of the system tests will be necessary once multihoming is complete.